js/src/apbct-public-bundle.js

```diff
@@ -2744,8 +2756,10 @@ function getJavascriptClientData(commonCookies = []) {
         ctCookiesTypeLocalStorage : ctCookiesTypeCookie;
     resultDataJson.apbct_pixel_url = ctPixelUrl !== undefined ?
         ctPixelUrl : ctCookiesPixelUrl;
-    if (resultDataJson.apbct_pixel_url && resultDataJson.apbct_pixel_url.indexOf('%3A%2F')) {
-        resultDataJson.apbct_pixel_url = decodeURIComponent(resultDataJson.apbct_pixel_url);
+    if (resultDataJson.apbct_pixel_url && resultDataJson.apbct_pixel_url == 'string') {
```

Comment:
Probably here you need to get the data type first?
... && typeof(resultDataJson.apbct_pixel_url) == 'string'